projects/plugins/jetpack/_inc/lib/admin-pages/class-jetpack-redux-state-helper.php

```diff
@@ -176,6 +176,7 @@ public static function get_initial_state() {
 				'hasUpdate' => (bool) get_theme_update_available( $current_theme ),
 				'support'   => array(
 					'infinite-scroll' => current_theme_supports( 'infinite-scroll' ) || in_array( $current_theme->get_stylesheet(), $inf_scr_support_themes, true ),
+					'webfonts'        => wp_is_block_theme() && function_exists( 'wp_register_webfont_provider' ) && function_exists( 'wp_register_webfonts' ),
```

Comment:
```suggestion
					'webfonts'        => WP_Theme_JSON_Resolver::theme_has_support() && function_exists( 'wp_register_webfont_provider' ) && function_exists( 'wp_register_webfonts' ),
```

Reply:
Technically, a theme could use a theme.json file to reveal the typography settings on blocks and make use of the Google Fonts module, without having any block templates, so I think this is a better check. But in practice I'm not sure if it will matter.

Reply:
Thanks for the tip. Done in 1146be46bea772c7623c73441fc59e145b3d2ffe